web/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -85,14 +108,18 @@ define(['angular', 'jquery'], function(angular, $) {
        * @param event {Object} The event object
        */
       $scope.moveWithKeyboard = function(widget, event) {
-        // get index independent of ng-repeat to avoid filter bugs
+        if ($scope.isToastDisplayed) {
+          return;
```

Comment:
Worth firing a supplemental toast or something to tell the user why their move attempt is being ignored?